packages/template/vite/tmpl/vite.main.config.mjs

```diff
@@ -1,4 +1,10 @@
 import { defineConfig } from 'vite';
 
 // https://vitejs.dev/config
-export default defineConfig({});
+export default defineConfig({
+  resolve: {
+    // Some libs that can run in both Web and Node.js, such as `axios`, we need to tell Vite to build them in Node.js.
+    browserField: false,
+    mainFields: ['module', 'jsnext:main', 'jsnext'],
```

Comment:
```diff
- mainFields: ['module', 'jsnext:main', 'jsnext'],
```

It's Vite default config.

https://vitejs.dev/config/shared-options.html#resolve-mainfields

<img width="465" alt="image" src="https://user-images.githubusercontent.com/26263658/232962591-41f2620b-9198-4f6a-aa70-e36b03e54acb.png">

Reply:
On the same page it also says:
> In future, `resolve.mainFields`'s default value will be `['browser', 'module', 'jsnext:main', 'jsnext']` and this option will be removed.

Reply:
- `mainFields` will always exist
  * https://github.com/vitejs/vite/blob/v4.2.2/packages/vite/src/node/plugins/resolve.ts#L64-L68
  * https://github.com/vitejs/vite/blob/v4.2.2/packages/vite/src/node/config.ts#L485
  * https://github.com/vitejs/vite/blob/v4.2.2/packages/vite/src/node/constants.ts#L11
- `browserField` will remove in futrue.
  * https://github.com/vitejs/vite/blob/v4.2.2/packages/vite/src/node/plugins/resolve.ts#L69-L73

Reply:
you're right. but that is `In future`. When I tested it, it still used the browser field if no browserField was present.